apps/astarte_pairing/lib/astarte_pairing_web/controllers/device_controller.ex

```diff
@@ -47,6 +49,18 @@ defmodule Astarte.PairingWeb.DeviceController do
       |> put_status(:created)
       |> put_view(CredentialsView)
       |> render("show_astarte_mqtt_v1.json", credentials: credentials)
+    else
+      {:error, reason} ->
+        Logger.info("Failure to create credentials.",
```

Comment:
grammar, pick your favorite


```suggestion
        Logger.info("Failure during credentials creation",
```


```suggestion
        Logger.info("Failed to create credentials",
```